pib_api/flask/schema/SQLAutoWithCamelCaseSchema.py

```diff
@@ -0,0 +1,16 @@
+from app.app import ma
+
+
+def camelcase(s):
```

Comment:
Added type hint. Since `s` stands for any generic string and the function is rather short I think in combination with the type hint it should be expressive enough now.